README.adoc

```diff
@@ -51,6 +51,48 @@ The link:https://youtu.be/B_2FXWI6CWg[30 minute video clip] below introduces Mul
 
 image:images/multibranch-pipeline.png[link=https://youtu.be/B_2FXWI6CWg]
 
+[#credential-binding]
+=== Git Credentials Binding
+
+The git plugin provides `Git Username and Password` binding that allows authenticated git operations over *HTTP* and *SSH* protocols using command line git in a Pipeline job.
+
+The git credential bindings are accessible through the `withCredentials` step of the link:https://plugins.jenkins.io/credentials-binding/[Credentials Binding] plugin.
+The binding retrieves credentials from the link:https://plugins.jenkins.io/credentials/[Credentials] plugin.
+
+==== Git Username and Password Binding
+
+This binding provides authentication support over *HTTP* protocol using command line git in a Pipeline job.
+
+Procedure::
+
+. Click the Pipeline Syntax _Snippet Generator_ and choose the `withCredentials` step, add Git Username and Password binding.
+. Choose the required credentials and Git tool name, specific to the generated Pipeline snippet.
+
+image:images/git-credentials-usernamepassword-binding-pipline-job.png[Git-Username-and Password-Binding-Pipeline-Job]
+
+NOTE: By default, two variable bindings, `GIT_USERNAME` and `GIT_PASSWORD`, bind with user’s credential i.e. username and password respectively, in a Pipeline jobs.
```

Comment:
This seems like an interesting implementation detail, but placing it in a NOTE gives it more emphasis that I think is justified.  The user should not need to do anything with either of these variables.  They are internal implementation details.
```suggestion
Two variable bindings are used, `GIT_USERNAME` and `GIT_PASSWORD`, to pass the username and password to `sh`, `bat`, and `powershell` steps inside the `withCredentials` block of a Pipeline job.
```